packages/mantine/src/components/table/table-header/Th.tsx

```diff
@@ -1,10 +1,19 @@
 import {ArrowDownSize16Px, ArrowUpSize16Px, DoubleArrowHeadVSize16Px} from '@coveord/plasma-react-icons';
-import {Group, UnstyledButton, useProps} from '@mantine/core';
-import {defaultColumnSizing, flexRender} from '@tanstack/react-table';
-import cx from 'clsx';
-import {AriaAttributes} from 'react';
-import ThClasses from './Th.module.css';
-import {SortState, ThProps} from './Th.types';
+import {BoxProps, Factory, Group, UnstyledButton, useProps} from '@mantine/core';
+import {CompoundStylesApiProps} from '@mantine/core/lib/core/styles-api/styles-api.types';
+import {defaultColumnSizing, flexRender, Header} from '@tanstack/react-table';
+import {AriaAttributes, ComponentType, ForwardedRef, SVGProps} from 'react';
+import {CustomComponentThemeExtend, identity} from '../../../utils';
```

Comment:
uber out of scope but I wonder if we shouldn't leverage more the `paths` param of the `compilerOptions` :joy: